Set default HZ to 1000 which is what most desktop users should still be using.

-ck

---
 kernel/Kconfig.hz |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-3.10-ck1/kernel/Kconfig.hz
===================================================================
--- linux-3.10-ck1.orig/kernel/Kconfig.hz	2013-07-09 17:29:01.183501909 +1000
+++ linux-3.10-ck1/kernel/Kconfig.hz	2013-07-09 17:29:01.287501905 +1000
@@ -4,7 +4,7 @@
 
 choice
 	prompt "Timer frequency"
-	default HZ_250
+	default HZ_1000
 	help
 	 Allows the configuration of the timer frequency. It is customary
 	 to have the timer interrupt run at 1000 Hz but 100 Hz may be more
